src/components/pages/ipfs/Error.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Header, Segment } from 'semantic-ui-react';
```

Comment:
we are going to probably remove those for tailwind so let's just put some basic div for now

Reply:
I still didn't update the style components to the tailwind